Fedoraware/TeamFortress2/TeamFortress2/SDK/Interfaces/Interfaces.cpp

```diff
@@ -14,6 +14,8 @@
 
 void CInterfaces::Init()
 {
+	using namespace I;
```

Comment:
why

Reply:
because it makes sense here since we are initializing the interfaces. There's no reason to write `I::` in this case